.github/ISSUE_TEMPLATE/02-bug-medium.yml

```diff
@@ -0,0 +1,61 @@
+name: Medium Severity Bug
+description: Used to report medium severity bugs in llama.cpp (e.g. Malfunctioning Features but generally still useable)
+title: "Bug: "
+labels: ["bug-unconfirmed", "medium severity"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Please include information about your system, the steps to reproduce the bug,
+        and the version of llama.cpp that you are using.
+        If possible, please provide a minimal code example that reproduces the bug.
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
```

Comment:
I don't think this needs to be here.

Reply:
it might be worth it if we want to send attachments or have a more private discussion (e.g. security discussion or something). Or maybe they don't check their github or something.

But anyway happy to remove it to keep things simple.

Reply:
That is rare enough (I don't remember it ever happening) that I don't think it needs to be in the default template, most people will just ignore it and it will add to the clutter.

It may be worth pointing that for sensitive security issues there is a better way to report them in https://github.com/ggerganov/llama.cpp/security